Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -2040,5 +2040,5 @@ Sollten Sie den Test in der App gelöscht haben, können Sie ihn aus dem Papierk
     <string name="fab_scanner_tip_title">QR-Code-Scanner</string>
     <!-- XTXT: UQS Tooltip subtitle -->
     <string name="fab_scanner_tip_subtitle">Scannen Sie Tests, Zertifikate und Check-ins mit diesem Scanner.</string>
-    <string name="home_eol_message">Vielen Dank für die Nutzung der Corona-Warn-App. Während der Pandemie wurden durch die Nutzenden der Corona-Warn-App rund 9 Millionen positive Testergebnisse geteilt. Durch diese Teilung positiver Testergebnisse konnten Personen schnell vor einem erhöhten Infektionsrisiko gewarnt werden, das ihnen sonst unerkannt geblieben wäre. Viele Gewarnte wurden anschließend positiv getestet. Die Corona-Warn-App hat dank Ihrer Mithilfe ihren Zweck der frühzeitigen Warnung erfüllt und die Beendigung von Infektionsketten unterstützt.\n\nAb dem 1. Juni 2023 wird die Corona-Warn-App nicht mehr weiterentwickelt. Sie können weiterhin auf Ihre bereits in der App gespeicherten Zertifikate und das Kontakt-Tagebuch zugreifen. Alle anderen Funktionen stehen nicht mehr zur Verfügung.\n\nWenn Sie wichtige Warnmeldungen für unterschiedliche Gefahrenlagen erhalten wollen, empfehlen wir Ihnen die <a href="https://www.bbk.bund.de/DE/Warnung-Vorsorge/Warn-App-NINA/NINA-Download/nina-download_node.html">Warn-App NINA des Bundesamts für Bevölkerungsschutz und Katastrophenhilfe (BBK)</a> (externer Link).</string>
+    <string name="home_eol_message">Vielen Dank für die Nutzung der Corona-Warn-App. Während der Pandemie wurden durch die Nutzenden der Corona-Warn-App rund 9 Millionen positive Testergebnisse geteilt. Durch diese Teilung positiver Testergebnisse konnten Personen schnell vor einem erhöhten Infektionsrisiko gewarnt werden, das ihnen sonst unerkannt geblieben wäre. Viele Gewarnte wurden anschließend positiv getestet. Die Corona-Warn-App hat dank Ihrer Mithilfe ihren Zweck der frühzeitigen Warnung erfüllt und die Beendigung von Infektionsketten unterstützt.\n\nAb dem 1. Juni 2023 wird die Corona-Warn-App in einen Ruhe-Modus versetzt. Sie können weiterhin auf Ihre bereits in der App gespeicherten Zertifikate und das Kontakt-Tagebuch zugreifen. Alle anderen Funktionen stehen nicht mehr zur Verfügung. Löschen Sie die App nicht; wenn diese reaktiviert oder anderweitig weiterentwickelt wird, werden Sie informiert.\n\nWenn Sie wichtige Warnmeldungen für unterschiedliche Gefahrenlagen erhalten wollen, empfehlen wir Ihnen die <a href="https://www.bbk.bund.de/DE/Warnung-Vorsorge/Warn-App-NINA/NINA-Download/nina-download_node.html">Warn-App NINA des Bundesamts für Bevölkerungsschutz und Katastrophenhilfe (BBK)</a> (externer Link).</string>
```

Comment:
Exact changes, as GH does not show this properly:
<img width="1265" alt="Bildschirmfoto 2023-05-15 um 12 09 16" src="https://github.com/corona-warn-app/cwa-app-android/assets/25054729/999d260b-299e-49be-b67f-f167d66b6921">